examples/preact/basic-file-based/src/routes/anchor.tsx

```
@@ -0,0 +1,209 @@
+import { createFileRoute } from '@tanstack/preact-router'
```

ESLint

[error] '@tanstack/preact-router' imported multiple times.

(import/no-duplicates)